src/styles/main.scss

```
@@ -1,3 +1,59 @@
+$calendarFont: arial;
```

It's good practise always specify fallback fonts.
```suggestion
$calendarFont: arial, sans-serif;
```

---

and I can't find where you use font in your css.

---

I used it in line number 40